config/v1/types_apiserver.go

```diff
@@ -0,0 +1,24 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// APIServer holds cluster-wide information about api-servers.  The canonical name is `cluster`
+type APIServer struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec APIServerSpec `json:"spec"`
```

Comment:
@deads2k this is also the place for the audit config, right?

Reply:
> @deads2k this is also the place for the audit config, right?

I think we'll have a separate top level audit.